lib/open_project/access_control/permission.rb

```diff
@@ -42,6 +42,7 @@ def initialize(name,
                      hash,
                      public: false,
                      require: nil,
+                     permissible_on: [],
```

Comment:
I'd opt to remove the default value here and make it mandatory to add this to a  permission definiton

Reply:
Ah, yes! I was thinking about adding a warning somewhere in case someone forgets about defining it but the error message raised by removing the default is much better. :)